libs/ledger-live-common/src/hw/actions/app.ts

```diff
@@ -27,7 +27,7 @@ import {
   getDerivationScheme,
   getDerivationModesForCurrency,
   runDerivationScheme,
-} from "../../derivation";
+} from "@ledgerhq/coin-framework/derivation";
```

Comment:
To be discussed, but I think ideally nothing in the `hw` part should depend on coin-framework, right?
Do you think something can be done now to remove the dependency, or should we leave for later? (then we need to add it to the list of TODOs)